### PR TITLE
Update documentation and metadata

### LINKED ISSUE #93: Add converter for OpenCell data
We need a converter for data generated by the [OpenCell pipeline](https://github.com/czbiohub/dragonfly-automation) such that it can be easily viewed and processed by downstream analysis pipelines.

The OpenCell pipeline generates image data and metadata in a custom format described [here](https://github.com/czbiohub/dragonfly-automation). The image data is usually too large to be opened as a ome-tiff dataset, which will load all positions, and cannot be opened by the iohub `MicromanagerOmeTiffReader` as it is missing the MM-specific Summary Metadata, see czbiohub/dragonfly-automation#12.

Two readers that currently exist for these datasets can be found in [dragonfly_automation/qc/opencell_qc.py](https://github.com/czbiohub/dragonfly-automation/blob/master/dragonfly_automation/qc/opencell_qc.py) and [opencell/imaging/images.py](https://github.com/czbiohub/opencell/blob/main/opencell/imaging/images.py)

### PR DESCRIPTION
**Pull request by ziw-liu**
Update documentation to reflect new features (#197).

Closes #93.

### CODE REVIEW

**Comment by ziw-liu** [comment]
@JoOkuma Can we consider DaXi to be supported?

**Comment by ziw-liu** [comment]
Also closes #15.

File: README.md [hunk 1]
```diff
@@ -11,9 +15,8 @@ used at the Biohub and in the broader imaging community.
 - OME-Zarr ([OME-NGFF v0.4](https://ngff.openmicroscopy.org/0.4/))
 - Micro-Manager TIFF sequence, OME-TIFF (MMStack), and NDTiff datasets
 - Custom data formats generated by Biohub microscopes
-  - Supported: Falcon (PTI), Dorado (ClearControl)
-  - WIP: Mantis, Dragonfly
-  - TBA: DaXi
+  - Supported: Falcon (PTI), Dorado (ClearControl), Dragonfly (OpenCell OME-TIFF)
+  - WIP: Mantis, DaXi
```
Comment (ieivanov) [inline README.md:19]: I think mantis is already supported, no? Are there other features we've planned to add before considering it "supported"?

Comment (ziw-liu) [reply README.md:19]: I was thinking of #142 and forgot that it has been solved. Moving this to 'supported'.

**Review by ieivanov — APPROVED** [review]

**Pull request merged**